HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -487,7 +488,7 @@ public IActionResult Eligible_Get()
     [HttpPost("eligible")]
     public async Task<IActionResult> Eligible_Post(EligibleViewModel viewModel)
     {
-        if (viewModel.CanContactByEmail is not YesOrNo.Yes && viewModel.CanContactByPhone is not YesOrNo.Yes)
+        if (viewModel.CanContactByEmail is YesOrNo.No && viewModel.CanContactByPhone is YesOrNo.No)

```

Comment:
This change stops the validation message coming up if the user hasn't selected a radio option at all - there are separate required validation messages on the radio controls.